spec/mail/message_spec.rb

```diff
@@ -1432,11 +1432,26 @@ def message_headers_should_match(message, other)
           part.body          = 'a' * 999
         end
         mail.encoded
-        
+
         expect(mail.parts.count).to eq(1)
         expect(mail.parts.last.content_transfer_encoding).to match(/7bit|8bit|binary/)
       end
 
+      # https://www.ietf.org/rfc/rfc1847.txt
+      # No encoding other than "quoted-printable", or "base64" is permitted for
+      # the body of a "multipart/signed" entity.
+      it "rfc1847" do
+        mail = Mail.new
+        mail.body << Mail::Part.new.tap do |part|
+          part.content_disposition = 'attachment; filename="test.asc"'
+          part.content_type  = 'application/pgp-signature'
+        end
+        mail.encoded
+
+        expect(mail.multipart?).to eq(true)
```

Comment:
`be_truthy`

Reply:
@jeremy Out of curiosity, do you prefer `be_truthy` just for consistency across the project, a personal taste, or to leave room for `multipart?` returning a truthy object that isn’t `true`?

Reply:
All of the above, yes! In essence, follow Ruby's lead that implicit coercion to boolean is a desirable default. Use explicit boolean values when explicitly necessary, which is rare.